docs/models.rst

```diff
@@ -184,10 +184,35 @@ For example:
 
 **Usage**
 
-Instantiate `InterpolatingComponent` and point it to the folder, define the unit and the target nside (same used by PySM).
+Instantiate `InterpolatingComponent` and point it to the folder, define the unit and the target :math:`N_{side}` (same used by PySM).
 It supports all `interpolation_kind` of :py:func:`scipy.interpolate.interp1d`, e.g. "nearest", "linear", "quadratic", "cubic"::
 
     cib = InterpolatingComponent(path="cib_precomputed_maps", input_units="MJysr", target_nside=nside, interpolation_kind="linear",
                              has_polarization=False, verbose=True)
 
-`Full example notebook <https://gist.github.com/zonca/08751497b040ec9d62ff5175573c786e>`
+`Full example notebook <https://gist.github.com/zonca/08751497b040ec9d62ff5175573c786e>`_
+
+WebSky
+======
+
+The Websky suite of simulated extragalactic component maps, determined from large scale structure light cone realizations and  based on Lagrangian perturbation theory, Peak Patch Lagrangian halo finding, and modeling of SZ and CIB effects, can be read into PySM as precomputed external fits files using InterpolatingComponent.  More information on the Peak Patch halo finding method can be found in `Stein, Alvarez, and Bond (2018) <https://arxiv.org/abs/1810.07727>`_, and selected maps and halo catalogs are available from the `Websky website <http://mocks.cita.utoronto.ca/websky>`_. For a description of the models as implemented for Simons Observatory, see the `SO Websky model repository <https://github.com/simonsobs/websky_model>`_.
+
+Websky maps generated as input for PySM are described below. Currently, cosmic infrared background at SO frequencies are available, with tSZ and kSZ coming soon.
```

Comment:
`tSZ` and `kSZ` are available, can @marcelo-alvarez update the docs?

Reply:
done